pkg/kubeclient/mur.go

```diff
@@ -0,0 +1,28 @@
+package kubeclient
+
+import (
+	crtapi "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+)
+
+const (
+	masterUserRecordResourcePlural = "masteruserrecords"
+)
+
+type masterUserRecordClient struct {
+	crtClient
+}
+
+type MasterUserRecordInterface interface {
+	Get(name string) (*crtapi.MasterUserRecord, error)
+}
+
+func (c *masterUserRecordClient) Get(name string) (*crtapi.MasterUserRecord, error) {
```

Comment:
could we add some documentation to our public apis?

Reply:
Done